docs/introduction/integration_test.rst

```diff
@@ -181,18 +181,34 @@ an additional reference can be added into the test_run decorator:
 
 In order to run only a subset of tests, an additional reference can be added to the test_run decorator:
 
-- variant : [optional] the variant can be defined like:
+- tag : [optional] the variant can be defined like:
 {"variant": ["variant2", "variant1"], "branch_level": ["daily", "nightly"]}
 
 Both parameters (variant/branch_level), will play the role of filter to fine
 tune the test collection and at the end ensure the execution of very specific tests subset.
 
-.. note:: branch_level parameter is also part of the CLI and both (variant/branch_level)
-    accept multiple values.
+.. note:: cli tags must be given in pairs. If one key has multiple values seperate them with a comma
 
 .. code:: bash
 
-    pykiso -c configuration_file --variant var1 --variant var2 --branch-level daily --branch-level nightly
+    pykiso -c configuration_file variant var1,var2 branch-level daily,nightly
```

Comment:
It would still be good to enforce the option names to start with -- (--branch-level, --variant, --other-tag, ...)
I think as it is the call is a bit confusing

Reply:
now every tag needs to begin with --  and only unknown kiso flags can be used as tags